frontends/ol-components/src/components/FormDialog/FormDialog.tsx

```diff
@@ -139,11 +131,9 @@ const FormDialog: React.FC<FormDialogProps> = ({
       isSubmitting={isSubmitting}
       className={className}
       PaperProps={paperProps}
+      actions={actions}
     >
-      <FormContent>
-        {children}
-        {footerContent}
```

Comment:
Anything that `footerContent` was used for can just be passed as part of the child.

(In a previous design, it needed to be separate)